deployment/base/rbac/master-clusterrole.yaml

```diff
@@ -12,11 +12,3 @@ rules:
   - patch
   - update
   - list
-- apiGroups:
```

Comment:
This `topology.node.k8s.io` changes don't align with the PR description

Reply:
Updated commit message